shell/platform/windows/keyboard_key_handler_unittests.cc

```diff
@@ -470,7 +470,7 @@ TEST(KeyboardKeyHandlerTest, AltGr) {
   EXPECT_EQ(handler.KeyboardHook(nullptr, VK_LCONTROL, kScanCodeControlLeft,
                                  WM_KEYDOWN, 0, false, false),
             true);
-  EXPECT_EQ(handler.KeyboardHook(nullptr, VK_LMENU, kScanCodeAltLeft,
+  EXPECT_EQ(handler.KeyboardHook(nullptr, VK_RMENU, kScanCodeAltLeft,
```

Comment:
Were these just all using the wrong side of the keyboard?

Reply:
Kind of. Previously the code that resolves side does not work very well and will send `LMENU` when right alt is pressed.